index.html

```diff
@@ -286,21 +326,36 @@ <h2 class="visually-hidden page-footer-title">Ссылки на социальн
 						<a class="footer-social-link footer-social-fb" src="/image/Facebook.svg"
 							href="https://www.facebook.com/htmlacademy">
 							<span class="visually-hidden">Facebook</span>
+							<svg width="9" height="18" viewBox="0 0 9 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+								<path d="M6.5 3H9V0H5.6C2.8 0 2 1.7 2 4.2V6H0V9H2V18H5V9H8.3L8.9 6H5V4.5C5 3.7 5.3 3 6.5 3Z"
+									fill="#444444" />
```

Comment:
вместо цвета нужно указать currentColor, чтобы можно было в CSS переопределять это свойство при взаимодействии

```suggestion
									fill="currentColor" />
```